securedrop/alembic/versions/b58139cfdc8c_add_checksum_columns_revoke_table.py

```diff
@@ -36,6 +36,10 @@ def upgrade():
     with op.batch_alter_table('submissions', schema=None) as batch_op:
         batch_op.add_column(sa.Column('checksum', sa.String(length=255), nullable=True))
 
+    with op.batch_alter_table('journalists', schema=None) as batch_op:
+        batch_op.add_column(sa.Column('first_name', sa.String(length=255), nullable=True))
+        batch_op.add_column(sa.Column('last_name', sa.String(length=255), nullable=True))
```

Comment:
since this isn't making it into 0.13.0 (feature freeze has passed), we'll need to move the migrations for this feature into a new migration targeted for 0.14.0

Reply:
implemented in 541d402f5bc19decaadc0d923a21a2ddeeb33884